### DISCUSSION TITLE
Welcome to the ChatGPTWizard Discussions!

### DISCUSSION THREAD

**Discussion by AliDehbansiahkarbon** [AliDehbansiahkarbon/ChatGPTWizard#1] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of Github. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with any ideas, suggestions, criticism, etc.

**Comment by CastleSoft** [comment]
What is the recommended way to install ChatGPTWizard when github has features not available in the GetItPackage manager ?

**Reply by AliDehbansiahkarbon** [reply]
Well, I'm doing some refinements on the latest version yet, I'm going to ask Embarcadero to update it after some updates and fixes.
But for time being, if you want to test that, the easy way is to have a clone of the repository on your machine, pull the latest version and follow the installation section in ReadMe. [here](https://github.com/AliDehbansiahkarbon/ChatGPTWizard#:~:text=Open%20the%20project%2C%20right%2Dclick%20on%20the%20project%20in%20the%20project%20manager%2C%20build%2C%20and%20install.)

An easy way to know how many new changes are there on Github without checking the Github directly is using a visual Git client like Fork(Find it [here](https://git-fork.com)), in this screenshot I can figure out that my local code is 31 commits behind the server in the "develop" branch and "develop_ali" has 31 new commit which are not pushed yet for instance.
![image](https://user-images.githubusercontent.com/5601608/230026181-1d7e8ac8-68b6-4269-a9f4-7c07d0884313.png)

**Reply by CastleSoft** [reply]
Thanks for the update, I'll give the suggestions a go...
